fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/socialmedia/NotifyAddChirp.java

```diff
@@ -6,7 +6,7 @@
 import com.google.gson.annotations.SerializedName;
 
 /** Data sent to broadcast AddChirp to the general channel */
```

Comment:
Is the documentation still correct ?

Reply:
Yes the protocol in itself still does the same thing, we changed the action name because we thought it was more intuitive. Hence why I changed the name of the class as well